.travis.yml

```diff
@@ -1,54 +1,51 @@
 #pretending we're C because otherwise ruby/rvm will initialize, even with "language: dm".
-language: c
+language: rust
+rust:
+- nightly
 sudo: false
 
 env:
-    global:
-    - BYOND_MAJOR="511"
-    - BYOND_MINOR="1363"
-    matrix:
-    - DM_MAPFILE="tgstation"
-    - DM_MAPFILE="metaclub"
-    - DM_MAPFILE="defficiency"
-    - DM_MAPFILE="taxistation"
-    - DM_MAPFILE="test_box"
-    - DM_MAPFILE="test_tiny"
+  global:
+  - BYOND_MAJOR="511"
+  - BYOND_MINOR="1363"
+  - ALL_MAPS="tgstation metaclub defficiency taxistation test_box test_tiny"
```

Comment:
Already were compiled in I'm not gonna change that.

Also checking for step vars is separate from this.